projects/cn0501/coraz7s/system_bd.tcl

```diff
@@ -7,12 +7,11 @@ source $ad_hdl_dir/projects/common/coraz7s/coraz7s_system_bd.tcl
 source ../common/cn0501_bd.tcl
 source $ad_hdl_dir/projects/scripts/adi_pd.tcl
 
-set mem_init_sys_path [get_env_param ADI_PROJECT_DIR ""]mem_init_sys.txt;
-
 #system ID
 ad_ip_parameter axi_sysid_0 CONFIG.ROM_ADDR_BITS 9
-ad_ip_parameter rom_sys_0 CONFIG.PATH_TO_FILE "[pwd]/$mem_init_sys_path"
+ad_ip_parameter rom_sys_0 CONFIG.PATH_TO_FILE "$mem_init_sys_file_path/mem_init_sys.txt"
 ad_ip_parameter rom_sys_0 CONFIG.ROM_ADDR_BITS 9
+
 set sys_cstring "sys rom custom string placeholder"
```

Comment:
Shouldn't `sys_cstring` be removed since the project doesn't have make parameters?